src/Core/Misc/PlogLogger.cpp

```diff
@@ -84,7 +86,19 @@ PlogLogger::PlogLogger() {
 }
 
 void PlogLogger::log(Core::ILogger::Severity logLevel, const std::string_view message) {
-    PLOG(PlogLogger::convertSeverity(logLevel)) << message;
+    const std::lock_guard<std::mutex> lock(this->mutex);
+    if (this->last_msg.empty()) {
```

Comment:
if last_msg is empty, then check to see if the new message is equal to that empty message? else if it isnt empty, set the last message to the new message? youve reversed the logic with the most recent commit

Reply:
Yeah, that's a major mistake, I'll get to fixing that.